src/instance/minecraft/chat/MuteChat.ts

```diff
@@ -7,23 +7,25 @@ import { EventType } from '../../../common/ApplicationEvent'
 export default {
   onChat: function (context: MinecraftChatContext): void {
     const regex =
-      /^(?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32}) has muted (the guild chat|(?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32})) for (\d)([dhms])/g
+      /^(?:\[[+A-Z]{1,10}] ){0,3}(\w{3,32}) has muted (?:\[[+A-Z]{1,10}] ){0,3}(the guild chat|\w{3,32}) for (\d)([dhms])/g
 
     const match = regex.exec(context.message)
     if (match != undefined) {
-      const username = match[2]
-      const muteTime = Number(match[4])
-      const muteSuffice = match[5]
+      console.log(match)
```

Comment:
Either use `logger` somewhere in `context` variable or delete that native `console.log()`.